version/version.go

```diff
@@ -2,11 +2,20 @@
 // describing the preflight project.
 package version
 
-import "fmt"
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/go-logr/logr"
+	"github.com/google/go-github/v57/github"
+	semvc "github.com/hashicorp/go-version"
+	"github.com/spf13/cobra"
+)
 
 var (
 	projectName = "github.com/redhat-openshift-ecosystem/openshift-preflight"
-	version     = "unknown"
+	version     = "0.0.1"
```

Comment:
Was this just changed so that other tests became valid?

Reply:
yes this was for tests

Reply:
Its not implicit ? Ok will add a comment 👍

Reply:
I don't think it's implicit, since it's been unknown or `0.0.0` locally forever.

Reply:
I would rather this remain its original value "unknown". We inject this at build time, and we only truly care about whether someone is using an outdated version in cases where they've used our release binaries, which have this value populated.

Which test needs this to change?

Reply:
I have tests comparing semver style versions thats why I changed it from unknown to 0.0.1. May be 0.0.0 is better ?